src/addremove.js

```diff
@@ -0,0 +1,40 @@
+import localStorageConst from './localstorage';
+
+const addremove = {
+  add: (title) => {
```

Comment:
* Kindly fix this functionality ` the property index set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal to 5).`. currently, when i add a new task it takes the index of the array eg(0,1,2,3) use the array length instead as stated in the project requirement.